packages/react/examples/codesandbox/patterns/blocks/ContentGroupCards/package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "ibmdotcom-react-parcel",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.html",
+  "scripts": {
+    "start": "parcel index.html --open",
+    "build": "parcel build index.html"
+  },
+  "dependencies": {
+    "@carbon/ibmdotcom-react": "1.7.0",
+    "@carbon/ibmdotcom-styles": "1.7.0",
+    "@carbon/icons-react": "10.11.0",
```

Just wondered if we can use newer version of `@carbon/icons-react`, for the sake of build speed. I believe it's compatible even with older version of `carbon-components-react` library.

```suggestion
    "@carbon/icons-react": "10.12.0",
```